### PR TITLE
Add healthcheck alert trigger.

### PR DESCRIPTION
**Pull request by Osmose**
This will let us test that alerts are actually being sent. @willkg for a dev review and @milescrabill to confirm this is suitable.

### CODE REVIEW

File: bec_alerts/jinja2/healthcheck.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+A healthcheck event was received in the bec-alerts queue. It worked!
+
+{{ issue.message }}
+{%- for frame in issue.stack_frames|reverse %}
+    at {{ frame.function }}({{ frame.module }}:{{ frame.lineNo|default('', True) }}:{{ frame.colNo|default('', True) }})
+{%- endfor %}
+
+This issue can be viewed in Sentry:
+
+https://sentry.prod.mozaws.net/operations/nightly-js-errors/issues/{{ issue.groupId }}/
```
Comment (willkg) [inline bec_alerts/jinja2/healthcheck.txt:10]: I wonder if there should be a "generic" message template that you can use for things. Just thinking out loud--no need to do anything here.

**Review by willkg — APPROVED** [review]
Looks good to me!

**Review by milescrabill — APPROVED** [review]
This will be penalized by Sentry's rate limiting as we discussed yesterday. Still, I think this is probably the right approach as it tests things end-to-end.

File: bec_alerts/jinja2/healthcheck.txt (hunk 1 above)
Comment (Osmose) [reply bec_alerts/jinja2/healthcheck.txt:10]: I was thinking of making the stack trace bits into a macro. Maybe the link-generation too.

Sounds like good steps for the third template.

**Pull request merged**